hack/go-test.sh

```diff
@@ -18,7 +18,6 @@ coverage=$1
 . ./hack/go-mod-env.sh
 
 if [[ -z ${CI} ]]; then
-    ./hack/go-vet.sh
```

Comment:
No because we run `make test` regularly and `vet` takes too much time to run. You can still run `vet` with `make vet`, just don't need to do it every time.